### DISCUSSION TITLE
Group multiple notifications into one email

### DISCUSSION THREAD

**Discussion by orbazan** [drakkan/sftpgo#1152] [Q&A]
Hello.
As described in the documentation i created the Upload Notification which works great. However our user case is that customers usually upload more than one file - it is not uncommon to upload 20-30 files at a time. Now that means that we also receive 20-30 separate emails, one email for every uploaded file which is not ideal to say the least - imagine suddenly receiving 30 emails...
Is there a way to group upload notifications into a single email? We used to use proftpd and we created a script that would run every 5 minutes and read the log and if there was any upload it would send an email something like 
"At ... customer X uploaded the following files:
File A
File B
File C
etc..."
Any solution would be greatly appreciated.
Thank you.

**Comment by drakkan** [comment]
Hello,

there is currently no way, we could group notifications and fire a single event when the user logs out, for example. A sponsorship would help. Alternatively you can parse the logs as you did with proftpd.
